src/components/Calculator.js

```diff
@@ -1,69 +1,61 @@
-import React, { Component } from 'react';
+import React, { useState } from 'react';
 import calculate from '../logic/Calculate';
 
-class Calculator extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      total: 0,
-      next: null,
-      operation: null,
-    };
-    this.handleCalculation = this.handleCalculation.bind(this);
-  }
+function Calculator() {
+  const [state, setState] = useState({
+    total: 0,
+    next: null,
+    operation: null,
+  });
```

Comment:
- [ ] Let us use ES6 syntax, kindly refactor with an arrow function instead of function

Reply:
Thank you for the feedback,I have implemented changes requested